src/web/lib/components/AppHeader/index.js

```diff
@@ -16,23 +18,32 @@ export const AppHeader = ({ hasExtension, appVersion = null }) => {
         <div className="app-header__icon" />
         <header>
           <h1>Themer</h1>
-          <h3>A <a href="https://testpilot.firefox.com" target="_blank"
-           rel="noopener noreferrer">Firefox Test Pilot</a> Experiment</h3>
+          <h3>
```

Comment:
nit: Not sure if the a11y linters called this out, but should this be an `h3` or an `h2`?

> _"Skipping heading ranks can be confusing and should be avoided where possible: Skipping heading ranks can be confusing and should be avoided where possible: Make sure that a `<h2>` is not followed directly by an `<h4>`, for example. It is ok to skip ranks when closing subsections, for instance, a `<h2>` beginning a new section, can follow an `<h4>` as it closes the previous section."_
>
> &mdash; via https://www.w3.org/WAI/tutorials/page-structure/headings/

Reply:
Not really a part of this PR or issue... that just got reformatted with pretty